libs/components/ag-grid/src/lib/styles/_base.scss

```diff
@@ -240,3 +240,12 @@ sky-ag-grid-wrapper.sky-ag-grid-layout-normal {
     }
   }
 }
+
+.ag-theme-sky-modern-light,
+.ag-theme-sky-modern-dark {
+  .ag-popup-editor {
+    --sky-background-color-input-box-group: var(
+      --sky-background-color-input-box-group-focused
```

Comment:
Let's get @Blackbaud-AdamFunderburk involved to see if we can come up with a more generic name for this CSS property since we usually don't have component-specific properties exposed to the public API.